kbdcfg.lua

```diff
@@ -81,6 +98,10 @@ function kbdcfg.bind()
         kbdcfg.widget = wibox.widget.imagebox()
     end
 
+    if kbdcfg.remember_layout then
```

Comment:
In this function you already know the number of layouts and you can add a check which value user have set for `kbdcfg.default_layout_index`. I suggest if index is incorrect we should set it equal to 1. Also it is necessary to update kbdcfg.current in case of wrong value.